tests/integration/test_fim/test_files/test_whodata_policy_change/data/test_cases/cases_whodata_policy_change.yaml

```diff
@@ -0,0 +1,17 @@
+- name: audit_policy_change_event_checker
+  description: Check when folders monitored w/ whodata mode, if audit policies change, fim changes to realtime mode
```

Comment:
Is `w/` make sense?

Reply:
fixed in https://github.com/wazuh/wazuh-qa/pull/3627/commits/3fdafde27f407d10ae203aed0114c566612f9801